packages/now-build-utils/src/detect-builders.ts

```diff
@@ -308,6 +313,24 @@ function validateFunctions(files: string[], { functions = {} }: Options) {
         };
       }
     }
+
+    if (func.includeFiles !== undefined) {
+      if (!Array.isArray(func.includeFiles) || !func.includeFiles.every(file => typeof file === 'string')) {
```

Comment:
You are missing the scenario when `includeFiles` is a string.

https://github.com/zeit/now/blob/b69f1761439b92e058aa7b9c30be0475073bcf38/packages/now-build-utils/src/types.ts#L45

Reply:
@styfle @leo I don't think we need to support that case here though, I'd rather just have one type instead of two types, which will be translated internally to the same one anyways.

I also don't think it's too bad if someone want's to migrate from `builds` to `functions` and just needs to wrap the string around `[` and `]`.

Reply:
If we want to make a breaking change here, lets go with a single `string` instead of an array.

Because the user can implement a glob that matches multiple directories like `{templates/**,views/**}`.

Reply:
Agreed!

We're currently enforcing the same across our new project settings. If the field has support for globs, it should be a string, since glob already adds support for multiple matches.